samples/qa-fonts/src/index.js

```diff
@@ -2,14 +2,13 @@ import {createRoot} from 'react-dom/client';
 
 import App from './App';
```

Comment:
I failed isomorphic pack on qa-fonts

```

Unable to generate prerender of app state HTML for en-US:
ReferenceError: document is not defined
    at Array.map (<anonymous>)
```

I guess there is problem on/qa-fonts/src/components/FontList/FontList.js
```
{document.fonts.check(font) ?
````

Reply:
🆗